src/parser/transform/transform_attach_database.cpp

```diff
@@ -1,18 +1,31 @@
 #include "parser/attach_database.h"
+#include "parser/expression/parsed_literal_expression.h"
 #include "parser/transformer.h"
 
 namespace kuzu {
 namespace parser {
 
+AttachOption Transformer::transformAttachOptions(CypherParser::KU_AttachOptionsContext& ctx) {
+    AttachOption option;
+    for (auto& attachOption : ctx.kU_AttachOption()) {
+        auto key = transformSymbolicName(*attachOption->oC_SymbolicName());
+        auto value = transformLiteral(*attachOption->oC_Literal())
```

Comment:
Normally you don't wanna perform any check/cast here. Just pass ParsedExpression and bind it later on